<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>544</width>
    <height>345</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>544</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuWelcome_Screen">
    <property name="title">
     <string>User</string>
    </property>
    <addaction name="actionRegister_User"/>
    <addaction name="actionLogout"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuGame">
    <property name="title">
     <string>Game</string>
    </property>
    <addaction name="actionNew_game"/>
    <addaction name="actionEnd_game"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionChange_Password"/>
   </widget>
   <addaction name="menuWelcome_Screen"/>
   <addaction name="menuGame"/>
   <addaction name="menuEdit"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionRegister_User">
   <property name="text">
    <string>Register User</string>
   </property>
  </action>
  <action name="actionLogout">
   <property name="text">
    <string>Logout</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionNew_game">
   <property name="text">
    <string>New game</string>
   </property>
  </action>
  <action name="actionEnd_game">
   <property name="text">
    <string>End game</string>
   </property>
  </action>
  <action name="actionChange_Password">
   <property name="text">
    <string>Change Password</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
